lib/floe/workflow/states/wait.rb

```diff
@@ -29,8 +29,8 @@ def start(input)
           context.output     = output_path.value(context, input)
           context.next_state = end? ? nil : @next
           wait(
-            :seconds => value_or_path(context, input, @seconds, :path => @seconds_path, &:to_i),
-            :time    => value_or_path(context, input, @timestamp, :path => @timestamp_path)
+            :seconds => @seconds_path ? @seconds_path.value(context, input).to_i : @seconds,
+            :time    => @timestamp_path ? @timestamp_path.value(context, input) : @timestamp
           )
```

Comment:
If I remember correctly. Before introducing this method I pulled this out into a separate method to avoid this complexity complaint

Let me know if you want to pull that out.